recipes-bsp/firmware-imx/firmware-imx_8.22.bb

```diff
@@ -16,8 +16,9 @@ do_install() {
     # SDMA Firmware section
     install -d ${D}${nonarch_base_libdir}/firmware/imx/sdma
     install -m 0644 ${S}/firmware/sdma/* ${D}${nonarch_base_libdir}/firmware/imx/sdma
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
+    # Comment these lines to use sdma-imx6q/7d.bin from here and not linux-firmware
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
```

Comment:
These are also changes that @otavio has rejected in the past.

Reply:
Not sure how this is resolved. Otavio wants these binaries from linux-firmware, not from this recipe, so I don't think you can take this change from meta-imx.

Reply:
@angolini please keep our version of this code.